src/MultiFormGoals/resources/css/common.scss

```diff
@@ -32,19 +26,27 @@
 
 .give-multi-form-goal-block__content {
 	display: grid;
-	grid-template-columns: 1fr 1fr;
+	grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+	grid-gap: 16px;
 	margin: 24px !important;
 	min-height: 250px;
-}
 
-.give-multi-form-goal-block__text {
-	padding-left: 16px;
-	display: flex;
-	flex-direction: column;
-	justify-content: center;
+	.give-multi-form-goal-block__text {
+		display: flex;
+		flex-direction: column;
+		justify-content: center;
+
+		h2 {
+			margin-bottom: var(--global--spacing-vertical);
```

Comment:
@henryholtgeerts this is the first that I recall seeing `--global--spacing-vertical` and a quick google search didn't seem helpful. Do you have any resources on this feature? Is it specific to SCSS?